Cargo.toml

```diff
@@ -19,8 +19,8 @@ geo = ["geojson"]
 name = "rs_es"
 
 [dependencies]
-reqwest = "0.9"
-log = "0.4"
+reqwest = "0.9.18"
```

Comment:
I'm not sure it is good to directly depend on `0.9.18` since this version breaks some integration (cf https://github.com/benashford/rs-es/issues/136)

depending on `0.9` makes it possible for the integrating binary to choose which `0.9` version they want

Reply:
which `reqwest` version fails to compile ?

maybe we can just require `^0.9.x` (where x is the first working version)

Reply:
This is an interesting case. The issue with #136 will only be fully solved in a future version of `rs-es` where we enable asynchronous operations, and in the meantime using it in asynchronous situations would probably be inadvisable even if we didn't have that error, as blocking will prevent things like Tokio running correctly.

I'm also not opposed to re-broadening the dependencies as a stop-gap measure.

Reply:
> maybe we can just require `^0.9.x` (where x is the first working version)

specifying `0.9.18` is equivalent to this, it is not `=0.9.18` :)

Reply:
my point would be to depend on at least `^0.9.16` since I have a sync binary that cannot use `0.9.18`

Reply:
Oh sorry I misunderstood, I was unable to get this to compile with 1.31 ([the minimum advertised](https://github.com/benashford/rs-es#building-and-installation)) on my machine with 0.9.16, if you have some luck then I think a PR is appropriate 👍

Reply:
0.9.16 of reqwest is quite a bit newer than 1.31 of Rust. So it's likely reqwest started using language features that wasn't available in older versions of Rustc, although that wouldn't explain why 0.9.18 did work on an older Rustc unless that same language feature stopped being used.

This is speculation I don't follow the development of reqwest.

Could the solution to this to update the minimum baseline of supported versions of rustc that `rs-es` supports. Then we could re-broaden the dependencies in `Cargo.toml` to keep everyone happy?

Reply:
I've just published version `0.12.2` which puts the dependencies back to as they were in `0.12.0`. I think this is safest in the meantime, as it's flexible enough that most users can tweak it as they wish.

All `0.9` versions of reqwest seem to work on rustc 1.36.0 (and probably the previous couple of versions), so if we assume all users are using a recent version of rustc then specifying the dependency as `0.9` is correct.

This does raise some questions about how compatible versions of the compiler should influence semantic versioning. I'm know sure what the done thing is in this case.